scripts/test-grep.sh

```diff
@@ -22,13 +22,21 @@
 #
 #############################################################################
 
-$1 2>&1 | tee $1.result
+$1 > $1.result 2>&1
+exit_status=$?
 
 if egrep -q 'ERROR: (LeakSanitizer|AddressSanitizer)' $1.result; then
    echo "SAN report detected"
    exit 1
 fi
 
+# Put error message into test-suite.log too, 
+# on passed cases, output is not redirected into test-suite.log
+cat $1.result
```

Comment:
This should be before the `if egrep -q ...`, as the script could exit, before the cat.

Reply:
Good point. I've fixed it.